parser-wrapper/interface-impl-v1/src/test/resources/org/mule/raml/implv1/api.raml

```diff
@@ -0,0 +1,249 @@
+#%RAML 0.8
```

Comment:
Any chance to reduce the api to a minimal one?  we should not check in customer apis